cmd/soroban-cli/Cargo.toml

```
@@ -16,6 +16,10 @@ default-run = "soroban"
 name = "soroban"
 path = "src/bin/main.rs"
 
+[package.metadata.binstall]
+pkg-url = "{ repo }/releases/download/v{ version }/{ name }-{ target }.{ archive-format }"
```

According to the cargo-bininstall docs, I think `archive-format` should be `archive-suffix`. Is that correct?

---

Actually, appears both work, but `archive-format` is "soft deprecated".

```suggestion
pkg-url = "{ repo }/releases/download/v{ version }/{ name }-{ target }.{ archive-suffix }"
```